magento/magento2
# Ratnesh 18901

### Description (*)

By applying this fix magento is unable to open forgot password page after customer login

### Fixed Issues (if relevant)

1. magento/magento2#18901: Forgot password form should not available while customer is logged in.

### Manual testing scenarios (*)

1. Goto forgot password link
2. Then goto customer login page and do successful login.
3. It should not redirect customer to forgot password page.

### Contribution checklist (*)
